pkgs/by-name/ar/argon/package.nix

```diff
@@ -0,0 +1,46 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  udev,
+  zstd,
+  stdenv,
+  darwin,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "argon";
+  version = "2.0.21";
+
+  src = fetchFromGitHub {
+    owner = "argon-rbx";
+    repo = "argon";
+    rev = version;
```

Comment:
nit
```suggestion
    rev = "refs/tags/${version}";
```

Reply:
Or newish
```suggestion
    tag = version;
```

Reply:
`tag` doesn't seem to work.

Reply:
> `tag` doesn't seem to work.

You would need to rebase on master first.

Reply:
(Not a problem, don't push again)